src/pages/people/Person.tsx

```diff
@@ -151,7 +156,18 @@ export default function Person(props: PersonProps) {
                 <Img style={addedStyles} src={img || defaultPic} />
               </div>
               <R style={{ width: hideActions ? 'calc(100% - 80px)' : 'calc(100% - 116px)' }}>
-                <Title style={{ fontSize: hideActions ? 17 : 20, margin: 0 }}>{owner_alias}</Title>
+                <Title
+                  style={{
+                    maxWidth: 200,
+                    whiteSpace: 'nowrap',
+                    overflow: 'hidden',
+                    textOverflow: 'ellipsis',
+                    fontSize: hideActions ? 17 : 20,
+                    margin: 0
+                  }}
```

Comment:
Instead of using the `style` prop we can instead make a styled component to make this code look a bit cleaner